transports/quic/Cargo.toml

```diff
@@ -0,0 +1,46 @@
+[package]
+name = "libp2p-quic"
+version = "0.6.0"
+authors = ["David Craven <david@craven.ch>", "Parity Technologies <admin@parity.io>"]
+edition = "2018"
+description = "libp2p-quic is a noise based quic implementation for rust-libp2p."
```

Comment:
Not only noise

Reply:
fixed in https://github.com/dvc94ch/rust-libp2p/pull/13